### ISSUE TITLE
Missing get_last_infrared_frame on Anaconda's Python3 32bit distribution

### ISSUE THREAD

**Issue by danilogr** [Kinect/PyKinect2#45]
I don't know if that's something you can solve on your side, but I just spent the last hour trying to understand why a couple of scripts that rely on PyKinect2 were failing.

It turns out that, despite initializing PyKinectV2 correctly with `PyKinectV2.FrameSourceTypes_Infrared`, `has_new_infrared_frame` was always returning `false`. Moreover, some scripts aborted because  `PyKinectV2Runtime` did not have the attribute `get_last_infrared_frame`.

I had previously installed pykinect2 through `pip install pykinect2`, so guess what `pip show pykinect2` returns?

```
Name: pykinect2
Version: 0.1.0
Summary: Wrapper to expose Kinect for Windows v2 API in Python
Home-page: https://github.com/Kinect/PyKinect2/
Author: Microsoft Corporation
Author-email: k4w@microsoft.com
License: MIT
Location: c:\programdata\anaconda3\lib\site-packages\pykinect2-0.1.0-py3.6.egg
Requires: numpy, comtypes
```

*Yes. Version 0.1.0.*

Solution? Cloning this repository and installing it through `easy_install`.
```
pip install comtypes
git clone https://github.com/Kinect/PyKinect2.git
python -m easy_install PyKinect2
```

*TL;DR:* If you are using Anaconda's python distribution on Windows, installing `pykinect2` through pip will result in installing an older version (`0.1.0`). To fix that, install it from the repository using `easy_install`
